org.civicrm.styleguide/templates/CRM/Styleguide/Page/StyleGuide.tpl

```diff
@@ -1,5 +1,5 @@
 {php}
-  $htmlBuilder = new CRM_Styleguide_HtmlBuilder();
+  $htmlBuilder = new CRM_Styleguide_HtmlBuilder($this->get_template_vars('styleguide'));
```

Comment:
Wouldn't it be better to instantiate the HtmlBuilder inside the Page controller and then assign it to the template?